packages/pixel_pusher/pixel_pusher.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+synopsis: "Control LED strips on Pixel Pusher hardware"
+description: """
+Light up LED strips by driving Pixel Pusher hardware modules over UDP/IP networks
+"""
+maintainer: "Michael Bacarella <michael.bacarella@gmail.com>"
+authors: ["Michael Bacarella <michael.bacarella@gmail.com>"]
+homepage: "https://github.com/mbacarella/pixel_pusher"
+bug-reports: "https://github.com/mbacarella/pixel_pusher/issues"
+dev-repo: "git+https://github.com/mbacarella/pixel_pusher.git"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
```

Comment:
```suggestion
depends: [
  "ocaml"
  "dune" {>= "1.11"}
```
According to your `dune-project` file, this  package requires at least dune 1.11

Reply:
lgtm, thanks for the hand-holding!